.storybook/main.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  stories: ['../stories/**/*.stories.@(ts|tsx|js|jsx)'],
+  stories: ['../stories/**/*.stories.@(ts|tsx|js|jsx)', '../src/**/*.stories.@(ts|tsx|js|jsx)'],
```

Comment:
we will not add stories in this path `'../stories/**/*.stories.@(ts|tsx|js|jsx)'` as each stories will be in each  component folder

Reply:
I will remove it and the stories folder